en/migrate-to-helm3.md

```diff
@@ -0,0 +1,155 @@
+tidb-operator   1               Tue Jan  5 15:28:00 2021        DEPLOYED        tidb-operator-v1.1.8    v1.1.8          tidb-admin
+```
+
+1. [Install Helm 3](https://helm.sh/docs/intro/install/).
+
+    Helm 3 takes a different approach from Helm 2 in configuration and data storage. Therefore, when you install Helm 3, there is no risk of overwriting the original configuration and data.
+
+    > **Note:**
+    >
+    > During the installation, do not let the CLI binary of Helm 3 overwrites that of Helm 2.  For example, you can name Helm 3 CLI binary as `helm3`. (All following examples in this document uses `helm3`.)
+
+2. Install [helm-2to3 plugin](https://github.com/helm/helm-2to3) for Helm 3.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm3 plugin install https://github.com/helm/helm-2to3
+    ```
+
+    You can verify whether the plugin is successfully installed by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm3 plugin list
+    ```
+
+    ```
+    NAME    VERSION DESCRIPTION
+    2to3    0.8.0   migrate and cleanup Helm v2 configuration and releases in-place to Helm v3
+    ```
+
+3. Migrate the configuration such as repo and plugins from Helm 2 to Helm 3:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm3 2to3 move config
+    ```
+
+    Before you migrate, you can learn about the possible operations and their consequences by executing `helm3 2to3 move config --dry-run`.
+
+    After the migration, the PingCAP repo is already in Helm 3:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm3 repo list
+    ```
+
+    ```
+    NAME    URL
+    pingcap https://charts.pingcap.org/
+    ```
+
+4. Migrate the releases from Helm 2 to Helm 3:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm3 2to3 convert tidb-operator
+    ```
+
+    Before you migrate, you can learn about the possible operations and their consequences by executing `helm3 2to3 convert tidb-operator --dry-run`.
+
+    After the migration, you can view the release corresponding to TiDB Operator in Helm 3:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm3 list --namespace=tidb-admin
+    ```
+
+    ```
+    NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                   APP VERSION
+    tidb-operator   tidb-admin      1               2021-01-05 07:28:00.3545941 +0000 UTC   deployed        tidb-operator-v1.1.8    v1.1.8
+    ```
+
+    > **Note:**
+    >
+    > If the original Helm 2 is Tillerless (Tiller is installed locally via plugins like [helm-tiller](https://github.com/rimusz/helm-tiller) rather than in the Kubernetes cluster), you can migrate the release by adding the `--tiller-out-cluster` flag in the command: `helm3 2to3 convert tidb-operator --tiller-out-cluster`
```

Comment:
```suggestion
    > If the original Helm 2 is Tillerless (Tiller is installed locally via plugins like [helm-tiller](https://github.com/rimusz/helm-tiller) rather than in the Kubernetes cluster), you can migrate the release by adding the `--tiller-out-cluster` flag in the command: `helm3 2to3 convert tidb-operator --tiller-out-cluster`.
```